connectd/usr/bin/connectd_startup_control

```diff
@@ -26,22 +26,29 @@ menuLoopA()
         printf "======================= Main Menu =======================\n"
         printf "                                                         \n"
         printf "    1) Set connectd_start to run at boot (systemd)       \n"
-        printf "    2) Disable connectd_start at boot                    \n"
-        printf "    3) Exit                                              \n"
+        printf "    2) Set connectd_start to run at boot (Sys-V)         \n"
```

Comment:
Will this change require any updates to our documentation?

Reply:
reverted to previous version of file